data/mods/Xedra_Evolved/effects/vampvirus.json

```
@@ -36,7 +36,7 @@
             "id": "EOC_REDUCE_VAMPVIRUS1",
             "condition": { "math": [ "u_effect_intensity('vampire_virus') == 2" ] },
             "effect": [
-              { "u_add_effect": "vampire_virus", "intensity": 1, "duration": "PERMANENT" },
+              { "u_add_effect": "vampire_virus", "duration": 0, "intensity": -1 },
```

```suggestion
              {
        "u_add_effect": "vampire_virus",
        "intensity": { "math": [ "u_effect_intensity('vampire_virus') - 1" ] },
        "duration": [ "72 h", "72 h" ]
      }
```
@SariusSkelrets so like this?

---

Since it's not the EOC to remove the effect, the duration needs to be permanent.

The rest is good.